### PR TITLE
query account details

### PR DESCRIPTION
**Pull request by babasahebk**
## Ticket
https://jira.bqsoft.com/browse/ATOM-8508

## Description
Query Account Details API

Request:
image

Response:
image
image
image
image

### CODE REVIEW

File: lib/conduit/plintron/parsers/query_account_detail.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    end
+
+    attribute :vms_pin do
+      translate_sim_status content_for "//VMS_PIN"
+    end
+
+    private
+
+    def translate_account_status(status)
+      ACCOUNT_STATUS_MAPPING[status] ? ACCOUNT_STATUS_MAPPING[status] : status
```
Comment (ufchrisg) [inline lib/conduit/plintron/parsers/query_account_detail.rb]:
```suggestion
      ACCOUNT_STATUS_MAPPING[status] || status
```

File: lib/conduit/plintron/parsers/query_account_detail.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    end
+
+    def translate_subscriber_status(status)
+      SUBSCRIBER_STATUS_MAPPING[status] ? SUBSCRIBER_STATUS_MAPPING[status] : status
```
Comment (ufchrisg) [inline lib/conduit/plintron/parsers/query_account_detail.rb]:
```suggestion
      SUBSCRIBER_STATUS_MAPPING[status] || status
```

File: lib/conduit/plintron/parsers/query_account_detail.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    end
+
+    def translate_sim_status(status)
+      SIM_STATUS_MAPPING[status] ? SIM_STATUS_MAPPING[status] : status
```
Comment (ufchrisg) [inline lib/conduit/plintron/parsers/query_account_detail.rb]:
```suggestion
      SIM_STATUS_MAPPING[status] || status
```

File: lib/conduit/plintron/views/query_account_detail.erb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<GET_ACCOUNT_DETAILS_REQUEST>
+<% if mdn %>
+<MSISDN><%= mdn %></MSISDN>
+<% elsif imsi %>
+<IMSI><%= imsi %></IMSI>
```
Comment (ufchrisg) [inline lib/conduit/plintron/views/query_account_detail.erb]: didn't plintron prefer IMSI if it was present?

Comment (babasahebk) [reply lib/conduit/plintron/views/query_account_detail.erb]: I thought for already active it is ok if we pass mdn but I just found their email saying best practice is to pass ICCID. I think I'll have to make change to use ICCID first at all places which I'll do it separate pull

Comment (ufchrisg) [reply lib/conduit/plintron/views/query_account_detail.erb]: That's what it was. I knew they preferred something over the others.

**Pull request merged**